### PR TITLE
Vastly improved error messages from the API

### LINKED ISSUE #218: Leverage API for more descriptive error messages
Currently the admin notice only displays the `code`. It should also display the `message` and separate the errors by lines instead of commas for readability.

### PR DESCRIPTION
**Pull request by bcampeau**
Fixes #218

### CODE REVIEW

File: admin/class-admin-apple-notice.php [hunk 1]
```diff
@@ -134,6 +134,22 @@ public static function show() {
 	 * @access private
 	 */
 	private static function show_notice( $message, $type ) {
+		// Format messages a little nicer
+		$message_array = explode( ':', $message );
+		if ( 2 === count( $message_array ) ) {
+			// If it's not 2, it's too unclear how to proceed.
+			// Try to split the second param on commas
+			$errors = explode( ',', $message_array[1] );
+			if ( count( $errors > 1 ) ) {
```
Comment (mslinnea) [inline admin/class-admin-apple-notice.php]: I think you mean `if ( count( $errors ) > 1 ) {`

**Comment by mslinnea** [comment]
just one thing, then 🚅

**Comment by bcampeau** [comment]
Fixed logic error from the initial review, updated to fix one more messaging issue and add release notes and a version bump for the official 1.1.6 release on Monday.

**Comment by jameswburke** [comment]
Looks good :bee:

**Pull request merged**